### ISSUE TITLE
תפסתי מומיה אבל היא נשארה שם

### ISSUE THREAD

**Issue by erelsgl** [PikaTeam/NumosAlphaVersion#7]
<div dir="rtl">
זרקתי נייר-חניטה על מומיה, וכנראה עניתי נכון (כי ראיתי למעלה שנוספו לי נקודות), אבל המומיה נשארה במקום, ולא קיבלתי שום חיווי על כך שהצלחתי.
כפי שלמדנו בהרצאה, חשוב לתת לשחקן משוב על הצלחה או כישלון - אם הוא הצליח, אז שיראה שמשהו קורה למומיה כך שיידע שהצליח; וגם אם הוא נכשל וענה לא נכון, שיראה משהו שיבהיר לו שהוא טעה.
</div>

**Comment by neyney10** [comment]
נכון, כרגע המומיות לא נעלמות עם תפיסתן, זהו סהכ חלק שעדיין לא מימשנו, אבל כן בתכנון המקורי.

אגב, שאלה, האם גודל הפונט קריא ונוח? גם בטקסט הדיבור למטה וגם בטקסט של הוראות כמו כניסה למקומות וכדומה.

**Comment by LioraPevzner** [comment]
סידרנו את זה, כרגע אם אתה פוגע במומיה אז במידה ואתה מצליח לענות על השאלה המתמטית היא נעלמת, אחרת היא נשארת וממשיכה לרדוף אותך.

**Comment by neyney10** [comment]
בנתיים עובדים על לעשות גם פידבק כלשהו בעת הצלחה או כשלון.

**Comment by LioraPevzner** [comment]
תיקנו את עניין הרזולוציות.
כמו כן הוספנו פידבקים עבור הצלחות במשימות, פירושו שאם מישהו מסיים את המשימה עד הסוף הוא מקבל מחיאות כפיים.
ניסינו באופן דומה להפעיל מחיאות כפיים\צליל של אכזבה בהתאם לתוצאת המשימה של המומיות אבל לא סיימנו לממש את זה, מאמינים שאומנם להגשה הקרובה זה לא יהיה מוכן אבל בעתיד בגרסאות המתקדמות יותר נדאג שזה יעבוד.

(הבעיה במשימה של המומיות הייתה ששמה הפעולה הייתה על פרפאב, ומסתבר שקצת בעייתי להפעיל סאונד על פראפב).
